wgpu-core/src/device/mod.rs

```diff
@@ -1040,12 +1041,16 @@ impl<B: GfxBackend> Device<B> {
             }
             pipeline::ShaderModuleSource::Wgsl(code) => {
                 profiling::scope!("naga::wgsl::parse_str");
-                // TODO: refactor the corresponding Naga error to be owned, and then
-                // display it instead of unwrapping
                 match naga::front::wgsl::parse_str(&code) {
                     Ok(module) => (None, Some(module)),
                     Err(err) => {
                         log::error!("Failed to parse WGSL code for {:?}: {}", desc.label, err);
+                        log::error!("{}:", err);
```

Comment:
err is already printed in the previous line

Reply:
Yeah I know. I just copied some lines over, without cleaning it up. At the moment this does nothing except print the error message twice :)